contracts/package.json

```diff
@@ -26,6 +26,7 @@
         "test:upgradable": "npx hardhat test test/upgradable.ts",
         "test:testnet": "npx hardhat test --network testnet",
         "test:previewnet": "npx hardhat test --network previewnet",
+        "test:mainnet": "npx hardhat test --network mainnet",
```

Comment:
Why do we need to test on the main net network? The primary requirements for the main net network should be the production tasks.

Reply:
Good point!